AM2RLauncher/AM2RLauncherLib/AM2RLauncherLib.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
This needs to stay as standard 2.0 (can probably be bumped to 2.1 tho) because windows launcher is framework and thus relies on it.
It's the reason why CI fails.

Reply:
Reverted back to `netstandard2.1`.

Reply:
this needs to be standard 2.0
2.1 strips framework support :(